README.md

```diff
@@ -140,9 +144,121 @@ In another terminal run the e2e tests
 # from
+export SPEC_FILE_PATTERN=./remote/specs/**/*.spec.js
+export SCREENSHOT_DIRECTORY=./remote/screenshots
 export BRANCH_ENV=master # or rc if branching from rc
 # export any custom external API URLs in env/custom.sh
-cd end-to-end-tests
+cd end-to-end-test
 yarn install
-yarn run test-webdriver-manager
+yarn run test-webdriver-manager # use `test-webdriver-manager-debug` for debugging of tests
 ```
+
+### Mount of frontend onto HTTPS backend
+A custom frontend can be tested against any backend in the web browser using a local node server (command `yarn run start`) and the `localdev` flag passed to th e browser (see section 'Check in cBioPortal context'). For remote backends that communicate over a HTTP over SSL (https) connection (e.g., cbioportal.org or rc.cbioportal.org), the frontend has to be served over SSL as well. In this case run `yarn run startSSL` in stead of `yarn run start`.
+
+## Run of `local e2e-tests`
+To enable e2e-tests on for features that depend on data that are not included in studies served by the public cBioPortal instance, cbioportal-frontend provides the `e2e local database` (refered to as _e2e-localdb_ or _local e2e_ in this text) facility that allows developers to load custom studies in any backend version used for e2e-tests. CircleCI runs the `e2e-localdb` tests as a separate job.
+
+Files for the local database e2e-tests are located in the `./end-to-end-test/local` directory of cbioportal-frontend. The directory structure of `./end-to-end-test/local` is comparable to that of the `./end-to-end-test/remote` directory used for e2e-tests against remote public cBioPortal instances.
+
+### Contexts for e2e-localdb tests
+E2e-tests can run in _local context_ (workstation for software development) or _CircleCI context_ (for continuous integration testing). On CircleCI, e2e-tests can be conducted on commits that are inside or outside the context of a pull request. These different contexts for _e2e-localdb_ tests are refered to below as _Local_, _CircleCi_ and _CircleCI+PR_ contexts, respectively.
+
+### Running e2e-localdb tests in _Local_ context for development
+Running of e2e-localdb tests in _Local_ context in essence follows the procedure as defined in `.circleci/config.yml`. In _Local_ context the e2e-localdb tests can be run in a dockerized environment (usefull for generating screenshots) or directly on the host sytem. The `end-to-end-test/local/runtime-config/run_container_screenshot_test.sh` script is included to run e2e-local tests in a dockerized environment in _Local_ context. The `end-to-end-test/local/runtime-config/run_local_screenshot_test.sh` is included to run e2e-local tests on the host system directly in _Local_ context. Prior to running the script several environmental variables should be set. For local development follow these steps:
+
+1. Add `export BACKEND=cbioportal:rc` (or other backend branch) to `/env/custom.sh`.
+2. Make sure the last commit is pushed to _origin_ repo (needed by jitpack).
```

Comment:
@pvannierop do we still need this?

Reply:
Yes, this procedure is used for test development (making spec.js files). I use it all the time.